Roslyakov Evgeny/hw1/index.html

```diff
@@ -29,30 +29,30 @@
                         <object type="image/svg+xml" data="img/graph.svg" class="graph"></object>
                     </div>
                     <div class="days">
-                        <div class="day">
-                            <div class="day__icon-inner">
+                        <ul class="day">
```

Comment:
Using list is a good idea, but I would've rather used `li ` with repeatable elements like 
```
<ul class="days">
  <li class="day">**and here all other staff with in <p> or <span> tags each**</li>
  <li>...</li>
</ul>
```

Reply:
I thought that the best practice will be making the list of equal element parts rather than the list of compound 'day' elements. But I will keep your remark in mind. Thanks for the review